src/gui/mzroll/mainwindow.cpp

```diff
@@ -3514,17 +3517,21 @@ void MainWindow::showFragmentationScans(float pmz) {
 
 	if (!fragPanel || fragPanel->isVisible() == false)
 		return;
+	
 	MassCutoff *massCutoff = getUserMassCutoff();
 
 	if (samples.size() <= 0)
 		return;
 	fragPanel->clearTree();
-	for (unsigned int i = 0; i < samples.size(); i++) {
-		for (unsigned int j = 0; j < samples[i]->scans.size(); j++) {
-			if (samples[i]->scans[j]->mslevel > 1
-					&& massCutoffDist(samples[i]->scans[j]->precursorMz, pmz,massCutoff) < massCutoff->getMassCutoff()) {
-				fragPanel->addScanItem(samples[i]->scans[j]);
+	for (auto sample : samples) {
+		for (auto scan : sample->scans) {
+			if (scan->mslevel < 2) continue;
+			if (massCutoffDist(scan->precursorMz,
```

Comment:
This is indented with tabs. Please switch to spaces

Reply:
Done.